CI/run_tests.sh

```diff
@@ -1,23 +1,29 @@
 #!/usr/bin/env bash
 
-sudo python ptf_nn/ptf_nn_agent.py \
+set -x
+
+sudo -E PATH=${PATH} python3 ptf_nn/ptf_nn_agent.py \
     --device-socket 0@tcp://127.0.0.1:10001 -i 0-1@veth0 \
-    &>/dev/null &
+    &
 
 sleep 5
 
-sudo python ptf_nn/ptf_nn_agent.py \
+sudo -E PATH=${PATH} python3 ptf_nn/ptf_nn_agent.py \
     --device-socket 1@tcp://127.0.0.1:10002 -i 1-1@veth3 \
-    &>/dev/null &
+    &
 
 sleep 5
 
-sudo python ptf_nn/ptf_nn_test_bridge.py -ifrom veth1 -ito veth2 \
-    &>/dev/null &
+sudo -E PATH=${PATH} python3 ptf_nn/ptf_nn_test_bridge.py -ifrom veth1 -ito veth2 \
+    &
 
 sleep 5
 
-sudo ptf --test-dir ptf_nn/ptf_nn_test \
+env
```

Comment:
Spurious?

Reply:
Removed in commit 30.  I don't know how I got that in there.